web-local/README.md

```diff
@@ -0,0 +1,17 @@
+# web-local
+
+This folder contains the local frontend implemented with TypeScript and [SvelteKit](https://kit.svelte.dev). 
+
+## Running in development
+
+To run the web server with an embedded runtime:
+
+1. In `web-local/build-tools/postinstall_runtime.sh`, edit the `RUNTIME_VERSION` constant with the commit hash of the runtime version you want to use.
+2. Run `npm run dev`
+
+To run both the web server and an external development runtime:
+
+1. Run `npm run dev -w runtime`
+2. Run `RILL_EXTERNAL_RUNTIME=true npm run dev`
```

Comment:
`RILL_EXTERNAL_RUNTIME=true` is not really needed. In dev mode it reuses the 8081 port if already in use.

Reply:
Cool, made that edit